### PR TITLE
feat(templates): add empty state for templates table

### PR DESCRIPTION
**Pull request by zouxuoz**

### CODE REVIEW

File: src/blocks/table/templates/templates.ejs [hunk 1]
```diff
@@ -56,8 +56,21 @@
             </table>
         </div>
     <% } else { %>
-        <div class="content">
-            Пустое состояние
+        <div class="content__message">
+            <div class="content__section">
+                <h3 class="content__messageTitle"><%- i18n('У вас пока нет шаблонов') %></h3>
+                <p><%- i18n('Создайте шаблон и используйте его для создания маркетинговых выпусков.') %>
```
Comment (radiansz) [inline src/blocks/table/templates/templates.ejs]: @trikashny need to write some texts

Comment (rosendi) [inline src/blocks/table/templates/templates.ejs]: @trikashny ping

**Comment by trikashny** [comment]
@radiansz [done](https://www.notion.so/85766212b51645df83760f8cc26153ce)

**Comment by zouxuoz** [comment]
@trikashny english version?

**Comment by radiansz** [comment]
LGTM :1234:

**Comment by trikashny** [comment]
@zouxuoz Here is english text 

> No saved templates.
> Templates make it fast and easy to create and run new campaigns and help to keep a unified style.
>

**Pull request merged**